src/main/java/org/dita/dost/writer/ForceUniqueFilter.java

```diff
@@ -117,4 +151,46 @@ private FileInfo generateCopyToTarget(final FileInfo srcFi, final int count) {
     public void setTempFileNameScheme(TempFileNameScheme tempFileNameScheme) {
         this.tempFileNameScheme = tempFileNameScheme;
     }
+    
+    /**
+     * Holds additional information about the parent for a topicref.
+     *
+     */
+    private static final class ParentTopicref {
+      /**
+       * The href value for the parent topic.
+       */
+      private URI href;
```

Comment:
By coding convention, the fields here should be `public final` and getters should be removed

Reply:
I made the requested changes